hypertrace-trace-enricher/hypertrace-trace-enricher-api/src/main/java/org/hypertrace/traceenricher/trace/util/StructuredTraceGraphBuilder.java

```diff
@@ -37,22 +41,25 @@ public static StructuredTraceGraph buildGraph(StructuredTrace trace) {
 
     if (shouldRebuildTraceEventsGraph || shouldRebuildTraceEntitiesGraph) {
       Instant start = Instant.now();
-      StructuredTraceGraph graph =
-          shouldRebuildTraceEventsGraph
-              ? StructuredTraceGraph.reCreateTraceEventsGraph(trace)
-              : StructuredTraceGraph.reCreateTraceEntitiesGraph(trace);
-      LOG.debug(
-          "Time taken in building TraceEventsGraph, duration_millis:{} for tenantId:{}",
-          Duration.between(start, Instant.now()).toMillis(),
-          trace.getCustomerId());
+      if (shouldRebuildTraceEventsGraph) {
+        cachedGraph.reCreateTraceEventsGraph(trace);
+      } else {
+        cachedGraph.reCreateTraceEntitiesGraph(trace);
+      }
+      if (LOG.isDebugEnabled()) {
```

Comment:
should this log be inside the if condition ?

Reply:
Didn't get your point, what changes are you suggesting in the log statement?

Reply:
it says building events graph but it could be either events or entities graph